packages/desktop-client/src/components/reports/spreadsheets/recalculate.ts

```diff
@@ -30,10 +28,10 @@ export function recalculate({
 }: recalculateProps) {
   let totalAssets = 0;
   let totalDebts = 0;
-  const monthData = months.reduce((arr, month) => {
+  const intervalData = intervals.reduce((arr, inter) => {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏
```suggestion
  const intervalData = intervals.reduce((arr, interval) => {
```